xml/System.Runtime.Intrinsics.X86/Popcnt+X64.xml

```diff
@@ -87,7 +87,7 @@
 
 ## Remarks
 
-This intrinsic is only available on 64-bit processes.
```

Comment:
This is actually not a typo. The requirement is that you are running in a 64-bit process. Running a 32-bit process on a 64-bit processor will still have this as `unsupported`.